tests/kernel/CMakeLists.txt

```diff
@@ -368,3 +368,44 @@ set_property(TEST
   "kernel/test_shuffle_long"
   "kernel/test_shuffle_ulong"
   APPEND PROPERTY LABELS "proxy")
+
+# disabled:
+# kernel/test_convert_type_16 : crash
+# all printf tests:
```

Comment:
Can you retest these with the latest LZ driver?

Reply:
I'm referring to the printf tests here, which I've been told have been fixed.

Reply:
there are still some printf bugs in the driver, but i also fixed some UB in pocl's printf test.